src/versionbits.h

```diff
@@ -59,6 +60,7 @@ static const int32_t nCFundAccSpreadVersionMask = 0x00004000;
 static const int32_t nCFundAmountV2Mask = 0x00010000;
 static const int32_t nCFundReducedQuorumMask = 0x00020000;
 static const int32_t nStaticRewardVersionMask = 0x00008000;
+static const int32_t nV451ForkMask = 0x00100000;
```

Comment:
0x00100000 = 0000 0000 0000 0000 0001 0000 0000 0000 0000 0000

which is version bit 20 if i counted right, the comment above says version bit 21. Not sure if it matters?

Reply:
you are right

Reply:
so i guess we just need to update the comment? everything else works off the bit mask anyway right?

Reply:
yes